ts/Core/Axis/PlotLineOrBand/PlotLineOrBand.ts

```diff
@@ -132,36 +142,21 @@ class PlotLineOrBand {
     public render(): (PlotLineOrBand|undefined) {
```

Comment:
I do not see much value in this chain concept when `undefined` is involved.

Reply:
It seems that the use case is to return `undefined` when the options are not valid. I'll update the docs.